gen/definitions/keychain.yaml

```diff
@@ -0,0 +1,24 @@
+---
+name: keychain
```

Comment:
`name` should be with capital letters -> `Keychain`

Also, just `Keychain` might be confusing as it is not the actual keychain -> `Keychain Manager`?